.travis.yml

```diff
@@ -20,10 +20,6 @@ matrix:
     - d: ldc-1.16.0
```

Comment:
ldc-1.17.0 Is it okay not to do a version-up here?

Reply:
The LDC upgrade is here: https://github.com/bpfkorea/agora/pull/283
And it needs a bit more work.

Reply:
Yes, I checked.